api/docs/oauth/index.yml

```diff
@@ -0,0 +1,64 @@
+openapi: 3.0.0
+servers:
+  - url: 'http://localhost:3000/spree_oauth'
+info:
+  version: 1.0.0
+  title: Spree OAuth Authentication
```

Comment:
I think we should explicitly tell that it’s OAuth 2.0 in case some newer version appears or someone will try to use oauth 1 client(yes, there are people using that. Like Twitter 🙄)